experimental/parser/legalize_file.go

```diff
@@ -222,6 +220,8 @@ func legalizePackage(p *parser, parent classified, idx int, first *ast.DeclPacka
 	legalizePath(p, taxa.Package.In(), decl.Path(), pathOptions{Relative: true})
 }
 
+var isOrdinaryFilePath = regexp.MustCompile("[0-9a-zA-Z./_-]*")
```

Comment:
Why move this down? The convention in most of Buf's codebases is to keep consts and vars at the top of the file (consistency makes them easier to find when you're not in an IDE).

Reply:
Closer to where it's used. I know that's the style, but for some reason I thought you'd said something about how you preferred to have stuff closer together? I can move it back.

(I wish go just had function-local globals lmao, this is such a caveman misfeature)